### PR TITLE
feat: add nix and vagrant-libvirt

### LINKED ISSUE #10: Add libvirt and vagrant plugin to the server we use for sandbox e2e test
Can we get libvirt and vagrant plugin `vagrant-libvirt` installed alongside the server that runs Virtualbox? I would like to run the same e2e test we run with Vagrant VirtualBox but with libvirt

### PR DESCRIPTION
**Pull request by rawkode**
closes #10 #11

### CODE REVIEW

**Review by gianarb — APPROVED** [review]

**Comment by ScottGarman** [comment]
@rawkode are you ready to merge this, or were you waiting for review from @mmlb? I'm reviewing some older PRs and noticed that once this is merged, work can continue on this Boots PR https://github.com/tinkerbell/boots/pull/159

**Comment by rawkode** [comment]
I was waiting on an OK from @mmlb. We can merge though

**Pull request merged**
